sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/aad/properties/AadAuthenticationProperties.java

```diff
@@ -82,42 +82,50 @@ public class AadAuthenticationProperties implements InitializingBean {
     private String redirectUriTemplate = "{baseUrl}/login/oauth2/code/";
 
     /**
-     * App ID URI which might be used in the "aud" claim of an id_token.
+     * App ID URI which might be used in the "aud" claim of an id_token. For instance, 'api://{applicationId}'.
+     * @see <a href="https://learn.microsoft.com/azure/active-directory/develop/security-best-practices-for-app-registration#application-id-uri">APP ID URI</a>
```

Comment:
Originally, `@` and `<` are not suggested to used in property java doc, because the java doc will be used to generate the appendix file in the reference doc of GitHub pages. Now GitHub pages will not be updated anymore, I think we can ignore this problem.

But we should update the reference doc in MS doc manually.

Reply:
This problem still exists: spring-configuration-metadata.json cannot render `@` and `<` nicely.

Reply:
Change to `refer to [doc](link)`  better?

Reply:
Still not display very well when it appears in the Intellij hint. How about doing like this:

> See Microsoft doc about APP ID URI for more detail: https://learn.microsoft.com/azure/active-directory/develop/security-best-practices-for-app-registration#application-id-uri

@saragluna FYI.